### DISCUSSION TITLE
Access to variables outside /scss/variables ?

### DISCUSSION THREAD

**Discussion by raplemie** [iTwin/iTwinUI#118] [Ideas]
I'm looking at creating a component for a `DropdownMenu` out of the `UserIcon` in the header (with Name / Email / Company...)

I can easily do that by simply adding a div as one of the generated menu item, (because there is no hover state or anything, I dont want to use the normal `MenuItem` (or `li`) in react) but I would like to have the same padding than other menu items below it, the issue is, this padding definition is `$iui-inputs-padding`, and this is defined in `scss/inputs/variables` (but not referenced in `scss/variables`.)

Would the expected solution to use directly the `scss/inputs/variables` or we intended to have only one entry point for variables consumption, in `scss/variables` ? 

Or, maybe the other variables are not meant to be part of the API and should be considered private ?

Thoughts ?

**Comment by mayank99** [comment]
> _the other variables are not meant to be part of the API and should be considered private_

Usually this is correct, however #97 is doing exactly what you're asking (moving variables from `inputs/` to global) 😄
